src/Components/PastMenu.js

```diff
@@ -0,0 +1,254 @@
+        },
+        {
+          image: cauliflowerLaab,
+          title: "Cauliflower Laab",
+          description: "Cauliflower Laab description",
+        },
+        {
+          image: shrimpPasteRelish,
+          title: "Shrimp Paste Relish",
+          description: "Shrimp Paste Relish description",
+        },
+        {
+          image: lemongrassJasmineRice,
+          title: "Lemongrass Jasmine Rice",
+          description: "Lemongrass Jasmine Rice description",
+        },
+      ],
+      desserts: [
+        {
+          image: smokedIceCreamWithCakeCroutons,
+          title: "Smoked Coconut Ice Cream with Cake Croutons",
+          description:
+            "Smoked Coconut Ice Cream with Cake Croutons description",
+        },
+      ],
+    },
+    "Nusantara Menu": {
+      description:
+        "From the south of Asia, we explore some lesser eaten foods in Singapore. The aromatic spices and flavours of this menu are top notch!",
+      starters: [
+        {
+          image: bandungSiomay,
+          title: "Bandun Siomay",
+          description: "Bandun Siomay description",
+        },
+        {
+          image: pepesTahuJamur,
+          title: "Pepes Tahu Jamur",
+          description: "Pepes Tahu Jamur description",
+        },
+        {
+          image: nasibal,
+          title: "Nasibal",
+          description: "Nasibal description",
+        },
+        {
+          image: sotoAyamIstimewa,
+          title: "Soto Ayam Istimewa",
+          description: "Soto Ayam Istimewa description",
+        },
+      ],
+      mains: [
+        {
+          image: bebekBetutu,
+          title: "Bebek Betutu",
+          description: "Bebek Betutu description",
+        },
+        {
+          image: ikanPesmol,
+          title: "Ikan Pesmol",
+          description: "Ikan Pesmol description",
+        },
+        {
+          image: babiRawon,
+          title: "Babi Rawon (8-12 pax)",
+          description: "Babi Rawon (8-12 pax) description",
+        },
+        {
+          image: plecingSayur,
+          title: "Plecing Sayur",
+          description: "Plecing Sayur description",
+        },
+        {
+          image: sambalHijau,
+          title: "Sambal Hijau",
+          description: "Sambal Hijau description",
+        },
+        {
+          image: vegetableSodhi,
+          title: "Vegetable Sodhi",
+          description: "Vegetable Sodhi description",
+        },
+        {
+          image: nasiLiwet,
+          title: "Nasi Liwet",
+          description: "Nasi Liwet description",
+        },
+      ],
+      desserts: [
+        {
+          image: notKlepon,
+          title: "Not Klepon",
+          description: "Not Klepon description",
+        },
+      ],
+    },
+  };
+  render() {
+    return (
+      <div className="row justify-content-center">
+        <h1>{this.props.menu}</h1>
+        <p className="col-lg-10 col-12">
+          {this.pastMenuDict[this.props.menu].description}
```

Comment:
Since menu can be null, you might access `.description` of undefined here, which would break your app